tools/test/check_syntax.sh

```diff
@@ -14,58 +14,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-if [ -z "$SYNTAX_FLAKE_ARGS" ]; then
-    SYNTAX_FLAKE_ARGS="--exclude=\"*.cfg\" bin/* lib/ tools/*.py"
+if [ -z "$SYNTAX_RUFF_ARGS" ]; then
+    SYNTAX_RUFF_ARGS="bin/ lib/ tools/*.py"
 fi
 
-if [ -z "$SYNTAX_PYLINT_ARGS" ]; then
-    SYNTAX_PYLINT_ARGS="lib/rucio/"
-fi
-
-if [ -z "$SYNTAX_PYLINT_BIN_ARGS" ]; then
-    SYNTAX_PYLINT_BIN_ARGS="bin/*"
-fi
-
-
-echo '==============================='
-echo 'Running flake8                 '
-echo '==============================='
-
-flake8 --ignore=E501,E722,W503 $SYNTAX_FLAKE_ARGS
-
-if [ $? -ne 0 ]; then
-    exit 1
-fi
-
-
 echo '==============================='
-echo 'Running pylint                 '
+echo 'Running Ruff                   '
 echo '==============================='
 
-pylint --rcfile=pylintrc --errors-only $SYNTAX_PYLINT_ARGS
+ruff check --extend-ignore=E501,E722 $SYNTAX_RUFF_ARGS
```

Comment:
Should we just include these ignored errors in the main Ruff config?

Reply:
I think you're seeing an older commit here! I fixed this in the next [commit](https://github.com/rucio/rucio/pull/7481/commits/8906c23585348bcc58e764d1d88f50e4a923b69e).